XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,69 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+		//This test is for populated clipboard only so we need to make sure there's something in Pasteboard
+		navigator.goto(URLBarOpen)
+		app.textFields["address"].typeText("www.google.com\n")
+        waitUntilPageLoad()
+        app.textFields["url"].press(forDuration:3)
+        app.tables.cells["menu-Copy-Link"].tap()
+		app.textFields["url"].tap()
```

Comment:
Indentation is not looking well...please see attached image...Not only here but all along the tests there are incorrect indentation...
<img width="439" alt="screen shot 2019-01-25 at 10 11 38" src="https://user-images.githubusercontent.com/1897507/51736214-7fce9780-2057-11e9-9430-4dfa9aa54c51.png">

Reply:
Thank you! That is so weird, I wonder if something is up with the text editor I'm using because it looks normal there but weird on Github. I will edit the file through XCode and make sure it is correct there :O